app/views/books/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2> <%= @book.title %> </h2>
+
+<h4>Average Rating: <%= @avg_rating%> </h4>
+
+<div id=high-rating>
+  <h4> Highest Rating: <%= @high_rating%> </h4>
+  <h5> What this reviewer had to say: </h5>
+  <p> <%= @high_review.body %> </p>
+  <h6> Contributed by: <%= @high_review.user.name %> </h6>
+</div>
+
+<h4> Lowest Rating: <%= @low_rating %></h4>
```

Comment:
It looks like you skipped the first user story. I would expect to be able to see all the reviews for a specfiic book with their body, user, and rating.